platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/activities/commands/SendCommand.java

```diff
@@ -90,4 +95,8 @@ private void sendActivity() throws Exception {
             }
         }
     }
+
+    public void setBundleContext(BundleContext bundleContext) {
```

Comment:
Several of the other command PRs included setters for varaibles using the osgi @Reference annotation. I looked through the documentation but couldn't see this specifically described so I tried removing it and it still seems to work. https://karaf.apache.org/manual/latest/#_console_3